tools/gatk4.wdl

```diff
@@ -0,0 +1,358 @@
+        File bam_index
+        File interval_list
+        File fasta
+        File fasta_index
+        File dict
+        String prefix = basename(bam, ".bam") + ".split"
+        Int modify_disk_size_gb = 0
+        Int ncpu = 8
+    }
+
+    Int disk_size_gb = ceil(size(bam, "GB") + 1) * 3 + ceil(size(fasta, "GB")) + modify_disk_size_gb
+
+    command <<<
+        set -euo pipefail
+
+        gatk \
+            SplitNCigarReads \
+            -R ~{fasta} \
+            -I ~{bam} \
+            -O ~{prefix}.bam \
+            -OBM true
+       # GATK is unreasonable and uses the plain ".bai" suffix.
+       mv ~{prefix}.bai ~{prefix}.bam.bai
+    >>>
+
+    output {
+        File split_n_reads_bam = "~{prefix}.bam"
+        File split_n_reads_bam_index = "~{prefix}.bam.bai"
+        File split_n_reads_bam_md5 = "~{prefix}.bam.md5"
+    }
+
+    runtime {
+        cpu: ncpu
+        memory: "24 GB"
+        disk: "~{disk_size_gb} GB"
+        container: "quay.io/biocontainers/gatk4:4.4.0.0--py36hdfd78af_0"
+        maxRetries: 1
+    }
+}
+
+task base_recalibrator {
+    meta {
+        description: "Generates recalibration report for base quality score recalibration."
+        external_help: "https://gatk.broadinstitute.org/hc/en-us/articles/360036897372-BaseRecalibratorSpark-BETA"
+        outputs: {
+            recalibration_report: "Recalibration report file"
+        }
+    }
+
+    parameter_meta  {
+        bam: "Input BAM format file on which to recabilbrate base quality scores"
+        bam_index: "BAM index file corresponding to the input BAM"
+        outfile_name: "Name for the output recalibration report."
+        fasta: "Reference genome in FASTA format"
+        fasta_index: "Index for FASTA format genome"
+        dict: "Dictionary file for FASTA format genome"
+        dbSNP_vcf: "dbSNP VCF file"
+        dbSNP_vcf_index: "dbSNP VCF index file"
+        known_indels_sites_VCFs: "List of VCF files containing known indels"
+        known_indels_sites_indices: "List of VCF index files corresponding to the VCF files in `known_indels_sites_VCFs`"
+        memory_gb: "RAM to allocate for task, specified in GB"
+        modify_memory_gb: "Add to or subtract from dynamic memory allocation. Default memory is determined by the size of the inputs. Specified in GB."
+        ncpu: "Number of cores to allocate for task"
+        use_original_quality_scores: "Use original quality scores from the input BAM. Default is to use recalibrated quality scores."
+    }
+
+    input {
+        File bam
+        File bam_index
+        String outfile_name = basename(bam, ".bam") + ".recal.txt"
+        File fasta
+        File fasta_index
+        File dict
+        File dbSNP_vcf
+        File dbSNP_vcf_index
+        Array[File] known_indels_sites_VCFs
+        Array[File] known_indels_sites_indices
+        Int memory_gb = 25
+        Int modify_disk_size_gb = 0
+        Int ncpu = 4
+        Boolean use_original_quality_scores = true
```

Comment:
If I recall prior discussion correctly, this makes the most sense as defaulting to `false` (and that's what the parameter meta says is the default), but the GATK WDL defaults it to `true`? I think we ignore the GATK WDL and set this as `false`.

Reply:
Good catch. Somehow I missed swapping that over.